src/components/common/GuideCard/GuideCard.jsx

```diff
@@ -148,7 +155,7 @@ const GuideCard = ({
                 <p className="mt-2 line-clamp-5 font-light">{guideText}</p>
               </div>
               <Link
-                to={`/cities/:id/${guideId}`}
+                to={generatePath(ROUTES.city, { id: guideId })}
```

Comment:
BUG: wrong link

Reply:
Yes, I changed the paths globally throughout the application, apparently I missed the path and replaced it incorrectly. Thanks for your attention.
Regarding ToastContainer, I read many articles, the problem is very popular, but the reasons are different for everyone. The solutions are also different. In my case, I needed to remove it from the guide card (I placed it in App.jsx, + I added a unique success and error ID to the toast.